code/src/reducers/nutrition.js

```diff
@@ -0,0 +1,118 @@
+import { createSlice } from '@reduxjs/toolkit'
+import fruitVeg from '../assets/fruitVeg.png'
+import dairy from '../assets/dairy.png'
+import freezer from '../assets/freezer.png'
+import meat from '../assets/meat.png'
+
+const initialState = {
+  list: {
+    shelves: [
+      {
+        name: 'Fruit & Veg',
+        reveal: false,
```

Comment:
// reveal- controls whether the products are hidden or displayed under the shelf